tests/tests_integration/test_oauth.py

```diff
@@ -70,31 +67,12 @@ def test_account_entity_is_as_expected(self):
                          resource_owner_secret=os.environ.get('MKM_ACCESS_TOKEN_SECRET'),
                          realm=url)
 
-        last_name = os.environ['MKM_ACCOUNT_LAST_NAME']
-        first_name = os.environ['MKM_ACCOUNT_FIRST_NAME']
-
-        expected_account_response = \
-            {
-                'account':
-                    {
-                        'name':
-                            {
-                                'lastName': last_name,
-                                'firstName': first_name
-                            }
-                    }
-            }
-
         r = requests.get(url, auth=auth)
 
         json_response = r.json()
 
         received_first_name = json_response['account']['name']['firstName']
         received_last_name = json_response['account']['name']['lastName']
 
-        expected_first_name = expected_account_response['account']['name']['firstName']
-        expected_last_name = expected_account_response['account']['name']['lastName']
-
-        self.assertEqual(received_first_name, expected_first_name, 'First name received is not as expected')
-
-        self.assertEqual(received_last_name, expected_last_name, 'Last name received is not as expected')
```

Comment:
This test is considerably different. Is it what you're looking for? Furthermore ``received_first_name`` is a string right? if it's a string, just use ``assert received_first_name`` without ``len()``. Few examples:
```
>>> a = ""
>>> assert a
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError
>>> a = None
>>> assert a
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError
>>> a = "a"
>>> assert a
# that's OK
```